android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -894,7 +895,12 @@ public boolean run(MwmActivity target)
         });
       }
       if (!isFirstStart && !NewsFragment.showOn(this))
-        LikesManager.INSTANCE.showDialogs(this);
+      {
+        if (ViralFragment.shouldDisplay())
+          new ViralFragment().show(getSupportFragmentManager(), "");
```

Comment:
Здесь без аргументов совсем - кода будет вместо одной строки - много)
> On Mar 18, 2016, at 5:40 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/MwmActivity.java <https://github.com/mapsme/omim/pull/2338#discussion_r56665646>:
> 
> > @@ -894,7 +895,12 @@ public boolean run(MwmActivity target)
> >          });
> >        }
> >        if (!isFirstStart && !NewsFragment.showOn(this))
> > -        LikesManager.INSTANCE.showDialogs(this);
> > +      {
> > +        if (ViralFragment.shouldDisplay())
> > +          new ViralFragment().show(getSupportFragmentManager(), "");
> Можно сделать как в BaseNewsFragment: вынести show() туда.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2338/files/49a98513afd524db41d893d7cec128cfb271218e..b0dcfabe82c6bad90154275d1b20f2651bbdf1b1#r56665646>